core/src/main/java/org/elasticsearch/plugins/RemovePluginCommand.java

```diff
@@ -56,7 +56,18 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
         execute(terminal, arg, env);
     }
 
-    // pkg private for testing
+    /**
+    * Remove the plugin named {@code plugin named}.
+    *
+    * @param terminal       the Terminal to use for input/output
+    * @param pluginName     the name of the plugin to remove.
+    * @param env            the environment for the local node, which may be used for the local settings and path
+    *
+    * @throws UserException if plugin name is null.
+    * @throws UserException if plugin directory does not exist. Returns command usage info on printing the plugin list
+    * @throws UserException if plugin bin directory does not exist
```

Comment:
This is not the right condition, a plugin bin directory is not required to exist.